src/main/java/core/basesyntax/service/impl/ReportWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class ReportWriterImpl implements ReportWriter {

```

Comment:
```suggestion
public class FileWriterImpl implements FileWriter {
```
in fact you can pass any data here and write to file